src/Commands.rb

```diff
@@ -2095,6 +2095,34 @@ def event_look_at?(ev, to, scope, metric = :square)
       end
       return x_axis && y_axis && (distance_between(metric, ev, to)<=scope)
     end
+    def event_look_towards_event?(ev1, ev2, scope)
+      event_look_towards?(ev1, event_x(ev2), event_y(ev2), scope)
+    end
+    def event_look_towards?(ev, x, y, scope)
+      ex, ey = event_x(ev), event_y(ev)
+      if event_direction(ev) == 8
```

Comment:
### [2] Juxtaposition of `if` structures invoking the same method

Instead of repeating the same conditional structure (`if; ;end`), and invoking the same method (`event_direction(ev)`) four times; wouldn't it be more suitable to use a `case/when` structure which actually invokes `event_direction(ev)` once ?

```ruby
case event_direction(ev)
when 8
  # [...]
when 2
  # [...]
when 4
  # [...]
when 6
  # [...]
end
```

### [3] Magic numbers

Instead of using number literrals (*such as* `2`, `4`, `6` *and* `8`) which doesn't explicitely tells the direction and thus impacts the legibility of this implementation; could we use some kind of constants such as :
```ruby
module Direction
  BOTTOM = 2
  LEFT   = 4
  RIGHT  = 6
  TOP    = 8
end
```

And then use them like that: `event_direction(ev) == Direction::BOTTOM` instead of `event_direction(ev) == 2` ?

Reply:
Both of your remarks are right. I'll change this.